src/contourFeature.js

```diff
@@ -1,93 +1,56 @@
 var inherit = require('./inherit');
-var feature = require('./feature');
+var meshFeature = require('./meshFeature');
 
 /**
  * Contour feature specification.
  *
  * @typedef {geo.feature.spec} geo.contourFeature.spec
  * @property {object[]} [data=[]] An array of arbitrary objects used to
- *  construct the feature.
+ *    construct the feature.
  * @property {object} [style] An object that contains style values for the
- *      feature.
- * @property {function|number} [style.opacity=1] The opacity on a scale of 0 to
- *      1.
- * @property {function|geo.geoPosition} [style.position=data] The position of
- *      each data element.  This defaults to just using `x`, `y`, and `z`
- *      properties of the data element itself.  The position is in the
- *      feature's gcs coordinates.
- * @property {function|number} [style.value=data.z] The contour value of each
- *      data element.  This defaults `z` properties of the data element.  If
- *      the value of a grid point is `null` or `undefined`, that point will not
- *      be included in the contour display.  Since the values are on a grid, if
- *      this point is in the interior of the grid, this can remove up to four
- *      squares.
+ *    feature.
+ * @property {number|function} [style.opacity=1] The opacity on a scale of 0 to
+ *    1.
+ * @property {geo.geoPosition|function} [style.position=data] The position of
+ *    each data element.  This defaults to just using `x`, `y`, and `z`
```

Comment:
very minor document issue (not sure if that matters when this will get rendered). Seems to have lot many blank spaces between the '.' and 'This'.

Reply:
There are two popular choices in typography, two spaces between sentences or one.  There are impassioned arguments for each side, and, since it is opinion, there is no strict right or wrong. 
 I use two spaces.  HTML is poor in that it collapses all spaces to a single space, losing the semantic difference between sentence separation and word separation, but in the source, this is still apparent.  In old typography manuals, it used to be argued that word separation should be 1/5 em and sentence separation should be 1/3 em, but that doesn't work in monospaced fonts.

Reply:
I see, I am fine with this as long as we are consistent. We could probably mention this somewhere so that a new developer follows the same.

Reply:
We need to update the docs in general to mentioned all of the desired behavior for jsdocs in general.